package.json

```diff
@@ -1,18 +1,17 @@
 {
   "name": "multiformats-website",
   "version": "1.0.0",
-  "description": "multiformats website",
+  "description": "Multiformats website",
   "main": "list-multihashes.js",
   "dependencies": {
+    "clean-css": "^5.3.2",
+    "less": "^4.1.3",
```

Comment:
I know it doesn't really matter, but for consistency I think these should be moved to `devDependencies`

Reply:
This one is now fixed.